apps/manga-reader/web/src/routes/routes.tsx

```diff
@@ -0,0 +1,106 @@
+import { lazyRouteComponent, RootRoute, Route } from "@tanstack/react-router";
+
+import Root from "../pages/root";
+
+const rootRoute = new RootRoute({
+  component: () => <Root />,
+});
+
+const authenticatedRoute = new Route({
+  getParentRoute: () => rootRoute,
+  component: lazyRouteComponent(async () => import("./authenticated")),
+  id: "authenticated",
+});
+
+const browseRoute = new Route({
+  getParentRoute: () => authenticatedRoute,
+  path: "browse",
+  component: lazyRouteComponent(async () => import("../pages/browse")),
+});
+
+const updatesRoute = new Route({
+  getParentRoute: () => authenticatedRoute,
+  path: "updates",
+  component: lazyRouteComponent(async () => import("../pages/updates")),
```

Comment:
**Suggestion:** Consider using a map or loop to create routes if they share similar properties to reduce redundancy.
```suggestion
const routes = ['browse', 'updates', 'settings', 'library', 'history', 'extensions'];
const routeObjects = routes.map(route => new Route({
  getParentRoute: () => authenticatedRoute,
  path: route,
  component: lazyRouteComponent(async () => import(`../pages/${route}`)),
}));
```